PR for green build in Pharo 10

hernanmd opened the pull request:
Currently in Pharo 10, only a few tests pass because Magritte uses a previous version of Grease (Grease:v1.8.x)

> 1727 ran, 126 passed, 0 skipped, 0 expected failures, 0 failures, 1601 errors, 0 passed unexpected

This PR update the Baseline to use Grease:v1.9.0 and all tests pass now:

> 1727 ran, 1727 passed, 0 skipped, 0 expected failures, 0 failures, 0 errors, 0 passed unexpected.

To reproduce it in a clean Pharo 10 image:

```smalltalk
EpMonitor disableDuring: [ 	
		Metacello new
 		baseline: 'Magritte';
		repository: 'github://hernanmd/magritte:pharo10-ready/source';
		load: 'Tests' ].
```

And run the Magritte tests

seandenigris commented:
Thanks! I haven't been following Grease closely. Why 1.9 in P10 and 1.8 in P9? Also, I think we should rename the `baseline330ForSqueak` platform attribute to `baseline330ForSqueakCommon` because it applies to both Pharo and Squeak.

hernanmd commented:
> 

Thanks Sean, I updated the PR with your suggestions, the 1.9 also works in Pharo 9 so there was no reason to use 1.8 anymore. I also renamed the method to #baseline330ForSqueakCommon
